NuGet.Config

```diff
@@ -54,12 +59,102 @@
       <package pattern="envdte80" />
       <package pattern="envdte90" />
       <package pattern="envdte90a" />
+      <package pattern="FluentAssertions" />
+      <package pattern="ILMerge" />
+      <package pattern="Lucene.Net" />
+      <package pattern="MessagePack" />
+      <package pattern="Microsoft.AspNetCore.App.Ref" />
+      <package pattern="Microsoft.Build" />
+      <package pattern="Microsoft.Build.*" />
+      <package pattern="Microsoft.Build.Artifacts" />
+      <package pattern="Microsoft.Build.Framework" />
+      <package pattern="Microsoft.Build.Locator" />
+      <package pattern="Microsoft.Build.Runtime" />
+      <package pattern="Microsoft.Build.Tasks.Core" />
+      <package pattern="Microsoft.Build.Utilities.Core" />
```

Comment:
all these that don't start with `Microsoft.`, are already mapped in `dotnet-public`, so mapping them here to vside makes these packages "less deterministic", since they now might be restored from two sources, rather than just one. MessagePack ***maybe*** has a special version just for VS, but the other ones are very, very unlikely to be used by anything else in VS, so it's not clear to me why mappings for these packaegs were added to vside, especially without removing the mapping from dotnet-public as well.

The MSBuild packages, I'm feeling very confident that there are not special versions in the vside feed, but like MessagePack, it's at least feasible that it could possibly have a VS specific build.